### DISCUSSION TITLE
Highlight the “Open” filter in the issues list

### DISCUSSION THREAD

**Discussion by j-f1** [community/community#5157] [Mobile] [answered]
By default, tapping on the "Issues" button on a repo home page takes you to a list of all open issues, just like on GitHub.com. However, while GitHub.com highlights the "X open" and includes `is:open` in the search box by default in order to teach users to find closed issues (or clear the filter to mix both open and closed issues), GitHub for iOS offers neither affordance. The way to show all issues or include closed issues is under the "Open" button which is not super clear. Additionally, while most other usages of those filter buttons across the app highlight the filter in blue when it is potentially excluding some results, the open/closed toggle is rendered in gray when filtering out closed issues.

My suggested fix is to highlight the open/closed toggle by default, and only render it in gray when both open and closed issues are shown in the filter results.

**Comment by Iron-Ham** [comment, marked as answer]
Thank you for the feedback. I'm going to open up an internal issue to track this.
